lib/Optimizer/IROptimizer/IROptimizer.cpp

```diff
@@ -30,21 +30,17 @@
 #include "llvm/Support/Casting.h"
 #include "llvm/Support/CommandLine.h"
 #include "llvm/Support/Debug.h"
+#include "llvm/Support/FileSystem.h"
```

Comment:
Do we still need these IO related headers in this file after you moved all the instrumentation logic into a separate file?

Reply:
I cleaned up the unused header includes.